app/components/Article/index.tsx

```diff
@@ -35,8 +38,25 @@ const ArticleFooter = (question: Question) => {
         <span>Did this page help you?</span>
 
         <CompositeButton className="flex-container">
-          <Action pageid={question.pageid} showText={true} actionType={ActionType.HELPFUL} />
-          <Action pageid={question.pageid} showText={true} actionType={ActionType.UNHELPFUL} />
+          <Action
+            pageid={question.pageid}
+            showText={true}
+            actionType={ActionType.HELPFUL}
+            onSuccess={() => setShowFeedback(true)}
+          />
+          <Action
+            pageid={question.pageid}
+            showText={true}
+            actionType={ActionType.UNHELPFUL}
+            onSuccess={() => setShowFeedbackForm(true)}
+          />
+          <span className={['action-feedback-text', showFeedback ? 'show' : ''].join(' ')}>
+            Thanks for your feedback!
```

Comment:
@melissasamworth  what will cause this to be hidden again? Should it disappear e.g. after 10s? Otherwise it will hang around and be annoying

Reply:
Yeah that works, disappears after 10s

Reply:
this also should probably go into the feedback form